packages/shared/wallet/src/hooks/use-connect.ts

```diff
@@ -1,89 +1,161 @@
+import { useCallback } from 'react';
 import { trackEvent } from '@notional-finance/helpers';
 import { useNotionalContext } from '@notional-finance/notionable-hooks';
 import { getNetworkFromId, TRACKING_EVENTS } from '@notional-finance/util';
-import { useConnectWallet, useSetChain } from '@web3-onboard/react';
-import { BigNumber, ethers } from 'ethers';
-import { useCallback, useEffect } from 'react';
+import MetaMask from '../images/meta-mask.svg';
+import WalletConnect from '../images/wallet-connect.svg';
+import CoinbaseWallet from '../images/coinbase-wallet.svg';
+import Safe from '../images/safe.svg';
+import { Connector, useConnect, useDisconnect } from 'wagmi';
```

Comment:
Can you rename the useConnect and useDisconnect imports to useWagmiConnect and useWagmiDisconnect